compiler/ast/parser.nim

```diff
@@ -104,9 +99,7 @@ proc simpleExprAux(p: var Parser, limit: int, mode: PrimaryMode): ParsedNode
 # implementation
 
 template prettySection(body) =
```

Comment:
Since the template is now redundant, could you remove it and it's usages? Feel free to disregard this comment if you're planning to re-use `prettySection` later on.

Reply:
Fixed, thanks for pointing this out. Now merging